currency-rates/arch-go.yaml

```diff
@@ -0,0 +1,53 @@
+version: 1
+threshold:
+  compliance: 100
+  coverage: 75
+dependenciesRules:
+  - package: "**.services.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.repo.**"
+        - "**.tools.**"
```

Comment:
Yes))) that's why adding a logger interface was a really bad idea from the start. In this case, the /internal/interfaces will only contain the logger interface because I use a different approach throughout the project. I don't see the need to add this interface to every package. So, the /tools package is an exception. I prefer to limit it to one package with redundant code rather than adding a second one